docs/pages/desktop-access/reference/configuration.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Desktop Access Configuration Reference
+description: Configuration reference for Teleport Desktop Access.
+---
+
+# Windows Desktop Service configuration
+
+`teleport.yaml` fields related to Desktop Access:
+
+```yaml
+# Main service responsible for Desktop Access.
+#
+# You can have multiple Desktop Access services in your cluster (but not in the
+# same teleport.yaml), connected to the same or different Active Directory
+# domains.
+(!docs/pages/includes/desktop-access/desktop-config.yaml!)
+```
+
+## Deployment
```

Comment:
This section might be a bit easier to parse if we split it into subsections: `### Direct mode` and `### IoT mode`.